### ISSUE TITLE
Unfetched attribute exception when mutating an entity that contains a read-only field

### ISSUE THREAD

**Issue by vyacheslav-pushkin** [Haulmont/jmix-graphql#85]
Reproducible by editing a `DatatypesTestEntity` instance. 

```
java.lang.IllegalStateException: Cannot get unfetched attribute [readOnlyStringAttr] from detached object com.company.scr.entity.test.DatatypesTestEntity-6a538099-9dfd-8761-fa32-b496c236dbe3 [detached].
```

Request payload:

```
{"operationName":"Upsert_scr_DatatypesTestEntity","variables":{"datatypesTestEntity":{"bigDecimalAttr":"11111","booleanAttr":null,"dateAttr":null,"dateTimeAttr":null,"doubleAttr":null,"integerAttr":null,"longAttr":null,"stringAttr":null,"timeAttr":null,"uuidAttr":null,"localDateTimeAttr":null,"offsetDateTimeAttr":null,"localDateAttr":null,"localTimeAttr":null,"offsetTimeAttr":null,"enumAttr":null,"associationO2Oattr":null,"associationM2Oattr":null,"associationM2Mattr":[],"intIdentityIdTestEntityAssociationO2OAttr":null,"integerIdTestEntityAssociationM2MAttr":[],"datatypesTestEntity3":null,"name":null,"readOnlyStringAttr":null,"id":"6a538099-9dfd-8761-fa32-b496c236dbe3"}},"query":"mutation Upsert_scr_DatatypesTestEntity($datatypesTestEntity: inp_scr_DatatypesTestEntity!) {\n  upsert_scr_DatatypesTestEntity(datatypesTestEntity: $datatypesTestEntity) {\n    id\n  }\n}\n"}
```

**Comment by DeNZoL63** [comment]
The exception has replaced with a more detailed one

**Comment by DartVerder** [comment]
Now displayed exception: `io.jmix.graphql.datafetcher.GqlEntityValidationException: Can't save entity to database`
[create_update_exception.txt](https://github.com/Haulmont/jmix-graphql/files/6491953/create_update_exception.txt)

**Comment by DartVerder** [comment]
The described message actual for the read-only entity (by security) - it will be fixed in #67.
For  the read only attribute (without setter) - message is looks similar: "Exception while fetching data (/upsert_Product) : Modifying read-only attributes is forbidden [name]"
